dev-aws/kafka-shared-msk/payment-platform/open-banking.tf

```diff
@@ -82,3 +82,17 @@ resource "kafka_topic" "payment-v1-events" {
     "cleanup.policy" = "delete"
   }
 }
+
+module "openbanking-apid" {
```

Comment:
I just realised that in terraform there is the best practice to name the resources with `_` instead of `-`: https://www.terraform-best-practices.com/naming

Reply:
@sbuliarca Do you know if I can update the name for the ones that were already created?

Reply:
Yes you can, but by default I think it will see them as new resources, thus destroying and recreating them. 
For keeping the resources you'd need [moved blocks](https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#moved-block-syntax)
If recreating is not ok, I can work together with you to get these migrated.

Reply:
Recreating should be fine. Thank you